roles/st2/tasks/main.yml

```diff
@@ -59,33 +59,33 @@
   tags: st2
 
 - name: Perform st2 version related operations
-  include: version.yml
+  import_tasks: version.yml
   tags: st2, version
 
 - name: Create and configure StackStorm system user
-  include: user.yml
+  import_tasks: user.yml
   tags: st2, user
 
 - name: Configure StackStorm authentication
-  include: auth.yml
+  import_tasks: auth.yml
   # Use st2_config.auth.enable setting if available, otherwise use st2_auth_enable
   when: (st2_config.auth|default({})).enable|default(st2_auth_enable)
   tags: st2, auth
 
 - name: Configure StackStorm st2.conf settings
   # Ansible nested loop to iterate through a hash of hashes
-  include: config.yml _conf_section_name={{ _conf_section.key }} _conf_options={{ _conf_section.value }}
-  with_dict: "{{ st2_config }}"
+  import_tasks: config.yml _conf_section_name={{ _conf_section.key }} _conf_options={{ _conf_section.value }}
```

Comment:
`include_tasks` when combined with a `loop`
To be forwards compatible with 2.7 (weird I know) 
Please move `_conf_*` vars to a `vars` block.
https://docs.ansible.com/ansible/devel/porting_guides/porting_guide_2.7.html#include-tasks-import-tasks-inline-variables

Reply:
not sure if this is going to be compatible with 2.5 (?)

Reply:
Yes it is compatible with 2.5 to change this. using the key=value notation works with `include` but has never been officially supported with `import_tasks` since it was introduced around 2.4 (I don't have the link to the GH issues I read about it). They just documented it with 2.7.

Reply:
Ah. I commented below. 

> ansible/ansible#32812 - dang this goes back to 2.4.1, so import_tasks doesn't support the var=value syntax at all.